src/D0_modelling_and_optimization.py

```diff
@@ -107,7 +107,7 @@ def warning_asset_type(asset, type, assetGroup):
 
```

Comment:
l. 103
>     if dict_values["simulation_settings"]["plot_nx_graph"]["value"] == True:

You could check for "display_networkx_graph" and "store_networkx_graph" here, or do so in the function *draw_graph*.